drivers/timer/mchp_xec_rtos_timer.c

```diff
@@ -217,6 +217,11 @@ static void xec_rtos_timer_isr(void *arg)
 {
 	ARG_UNUSED(arg);
 
+#ifdef CONFIG_EXECUTION_BENCHMARKING
+	extern void read_timer_start_of_tick_handler(void);
```

Comment:
I think this ought to be exposed in a header, perhaps under include/debug/ and namespaced with z_. I guess this PR didn't introduce this though.

Reply:
All the  `read_timer_{start,end}_of_*` in `arch/common/timing_info_bench.c` are not exposed.